### PR TITLE
Update the game over message

### PR DESCRIPTION
**Pull request by dgbyer**
Update the game over message so people know how to play again!

### CODE REVIEW

**Review by dgbyer — COMMENTED** [review]
Pretty straight forward.

File: index.html [hunk 1]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
 
-<h2 hidden>Game over</h2>
+<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
```
Comment (leadass) [inline index.html:50]: adding comment to test

File: index.html [hunk 2]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
 
-<h2 hidden>Game over</h2>
```
Comment (dgbyer) [inline index.html:50]: suggestion

File: index.html [hunk 3]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
```
Comment (dgbyer) [inline index.html:48]:
```suggestion
</table>
```
suggestion for table

File: index.html (hunk 1 above)
Comment (dgbyer) [inline index.html]:
```suggestion
<h2 hidden>Game over, play again 🧑‍💻 🤖!</h2>
```
removed 'refresh to'

**Pull request merged**